src/include/collectives.h

```diff
@@ -123,11 +123,11 @@ extern __device__ void NCCL_ONERANK_REDUCE_NAME(PreMulSum, double)(struct ncclWo
 #define REDUCESCATTER_SLICESTEPS 2
 #define REDUCESCATTER_CHUNKSTEPS 4
 #define BROADCAST_SLICESTEPS 1
-#define BROADCAST_CHUNKSTEPS 2
+#define BROADCAST_CHUNKSTEPS 4
 #define REDUCE_SLICESTEPS 1
-#define REDUCE_CHUNKSTEPS 2
+#define REDUCE_CHUNKSTEPS 4
 #define SENDRECV_SLICEFACTOR 1
-#define NCCL_MAX_SLICE_PER_CHUNK 2  // max value for CHUNKSTEPS/SLICESTEPS, must accord with above
+#define NCCL_MAX_SLICE_PER_CHUNK 4  // max value for CHUNKSTEPS/SLICESTEPS, must accord with above
 #define ALLTOALL_PIVOT_SLICESTEPS 2
```

Comment:
Each collectives may use different slice and chunk steps depends on performance tuning. NCCL_MAX_SLICE_PER_CHUNK define the max value of slices per chunk. Since these constants are originated from NCCL, changing definitions will result in source code changes, which makes future merge harder.